python/dgl/distributed/graph_partition_book.py

```diff
@@ -408,101 +494,238 @@ class RangePartitionBook(GraphPartitionBook):
         partition id of current partition book
     num_parts : int
         number of total partitions
-    node_map : tensor
-        map global node id to partition id
-    edge_map : tensor
-        map global edge id to partition id
+    node_map : a dict of tensors
+        map global node id to partition id for each node type.
+        The key of the dict is the node type and the value is a 2-column matrix.
+        Each row represents the Id range of a partition, the first element is the starting Id
+        and the second element is the ending Id.
```

Comment:
After reading the implementation, I found my understanding maybe wrong. Look like the `node_map` stores global ID ranges instead of local ID ranges. If so, pls doc it clearly.

Reply:
yes, `node_map` stores the global IDs. I have corrected it.

We cannot store the number of nodes for each parition. these are global IDs. The ID ranges of a node type in different partitions are connected.